sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/RecognizeContent.java

```diff
@@ -41,14 +41,15 @@ public static void main(final String[] args) throws IOException {
         byte[] fileContent = Files.readAllBytes(sourceFile.toPath());
         InputStream targetStream = new ByteArrayInputStream(fileContent);
 
-        SyncPoller<OperationResult, IterableStream<FormPage>> recognizeLayoutPoller =
+        SyncPoller<OperationResult, List<FormPage>> recognizeLayoutPoller =
             client.beginRecognizeContent(targetStream, sourceFile.length(), FormContentType.IMAGE_JPEG);
 
-        IterableStream<FormPage> layoutPageResults = recognizeLayoutPoller.getFinalResult();
+        List<FormPage> contentPageResults = recognizeLayoutPoller.getFinalResult();
```

Comment:
since you're changing the name of layoutPageResults -> contentPageResults, I think it's a good idea to change recognizeLayoutPoller -> recognizeContentPoller